test/integration/puppetlabs/services/master/tasks_int_test.clj

```diff
@@ -0,0 +1,101 @@
+(ns puppetlabs.services.master.tasks-int-test
+  (:require [clojure.test :refer :all]
```

Comment:
Of these requires, I don't think `clojure.string`, `puppetlabs.trapperkeeper.testutils.bootstrap`, `puppetlabs.trapperkeeper.testutils.webserver`, `puppetlabs.services.master.master-core`, `puppetlabs.services.protocols.jruby-puppet`, `clojure.tools.logging`, or `puppetlabs.services.jruby.jruby-puppet-testutils` are used.